src/PrestaShopBundle/Resources/views/Admin/Improve/Design/MailTheme/Blocks/list_mail_theme_layouts.html.twig

```diff
@@ -90,6 +93,10 @@
                               <i class="material-icons">subject</i>
                               {{ 'Text'|trans({}, 'Admin.Design.Feature') }}
                             </a>
+                            <a class="btn tooltip-link dropdown-item" href="{{ mailUrl }}">
+                              <i class="material-icons">email</i>
+                              {{ 'Send a test email'|trans({}, 'Admin.Design.Feature') }}
```

Comment:
It already exists in _Admin.Advparameters.Feature_, let's avoid duplication and use the same key. ;-)

Reply:
Ok, it's juste that it might be weird to mix the domains in the same page no?

Reply:
Well, it is often the case actually, and I have always heard that the good practice was to avoid duplicating same translation strings so I guess it is better to use _Admin.Advparameters.Feature_ this time.